lxqt-panel/panel/man/lxqt-panel.1
.TH lxqt-panel "1" "2015-11-05" "LXQt 0.10.0" "LXQt Desktop Panel Module"
.SH NAME
lxqt-panel \- Desktop panel for \fBLXQt\fR: The Lightweight Qt Desktop Environment
.SH SYNOPSIS
.B lxqt-panel
.br
.SH DESCRIPTION
This module adds a panel, with optional plugins, to the desktop.
.SH BEHAVIOR
The panel can be run independently of \fBLXQt\fR, autostarted at logon, and have
multiple instances. A horizontal bottom panel shows by default on the desktop,
but the alignment, size, autohide, transparency (requires compositor), and other
attributes are user configurable.
.P
The panel is comprised of plugins which provide a visual widget; like the menu,
clock, or volume. They can be added or removed in the panel Widget settings.
.P
Several plugins are loaded by default; the desktop menu and windows workspaces
are also managed here.
.SH CONFIGURATIONS
Right-click over any plugin to reach the panel Configure settings option, or
that of each respective plugin.
.SH "REPORTING BUGS"
Report bugs to https://github.com/lxqt/lxqt/issues
.SH "SEE ALSO"
.\" any module must refer to the session application, for module overview and initiation
\fBstartlxqt.1\fR  LXQt session initialization and launch script (e.g. in \fB.xinitrc\fR)
.P
\fBlxqt-session.1\fR  LXQt \fIoverview\fR and complete session environment
.P
\fBlxqt-config-session.1\fR  LXQt default and autostart applications settings,
plus environment settings
.P
\fBlxqt-config.1\fR  LXQt settings Configuration Center interface
.P


CHANGELOG

lxqt-build-tools-0.6.0 / 2018-01-25
===================================

  * Bump version to 0.6.0
  * Don't use automatic string conversions
  * Introduced lxqt-transupdate for translation updates
  * Improved cmake scripting
    - Set cmake_minimum_required to 3.1.0
    - Consider QT_SELECT - some distributions use this tool
    - Drop 0X and 11 support, require c++14
    - Remvoed QUIET from any finds
    - Cleaned up LXQtTranslateTs module
    - Removed translation pulling and cleanup external repos
      after translations moved back to their original repos
    - Remove link-time optimization for gcc since it breaks
      qobject_cast() in some cases.
    - Removed obsolete FindFm module

lxqt-build-tools-0.5.0 / 2018-05-21
===================================

  * Bump version to 0.5.0
  * Prevent in-source builds
  * Adds an PreventInSourceBuilds module
  * http --> https
  * Fixed some LXDE mentions
  * LXQtCompilerSettings: Set CMAKE_C_VISIBILITY_PRESET to hidden
  * Don't allow Qt foreach/Q_FOREACH
  * Update CMakeLists.txt
  * Require Qt 5.7.1
  * Adds a FindXKBCommon CMake module
  * Adds ECMFindModuleHelpers CMake module
  * Don't export github templates

0.4.0 / 2017-09-22
==================

  * Release 0.4.0: Update changelog
  * Bump version to 0.4.0
  * Added needed versions
  * remove version set from LXQtConfigVars.in
  * Move LXQt version to liblxqt
  * Fixes a FTBFS with superbuild mode
  * Adds a FindExif module
  * LXQtCompilerSettings: Fixes a typo in diagnostics-color
  * Adds (Ninja) colored diagnostics for GCC (>=4.9)
  * Enable strict iterators for debug builds

0.3.2 / 2017-01-14
==================

  * Release 0.3.2: Update changelog
  * Bump version to 0.3.2 - finally the build tools are arch: all (#15)
  * Warnings (#16)
  * Remove arched bits from version.cmake-file (#14)

0.3.1 / 2017-01-01
==================

  * Release 0.3.1: Update changelog
  * Bump LXQT_PATCH_VERSION and build tools version (#13)
  * 32 64 bit ness (#12)
  * Use "gcc-ar" and "gcc-ranlib" tools when building static libs with gcc link-time optimization. (#11)

0.3.0 / 2016-12-06
==================

  * Release 0.3.0: Update changelog
  * Adds an FindXCB CMake module
  * FindUDev: Add VERSION_STRING support
  * FindUDev: Set REQUIRED_VARS
  * FindUdev: Feed pkg_modules_modules results to find_library/path
  * FindUDev: Adds Feature Summary
  * Adds FindUDev module
  * Enables colored diagnostics for CLang/Ninja combination
  * Disable link-time optimization for clang++/llvm since after some testing it breaks QObject signal/slot sometimes. * Remove trailing spaces.
  * Use different link-time optimization compiler options for gcc and clang and move -Bsymbolic flags to linker flags.
  * Add more linker options to turn on linker-time optimizations for gcc & clang.
  * Set CMAKE_BUILD_TYPE to Release by default if it's value is not set.
  * FindGLIB: Improve gio-unix handling
  * Adds a FindGLIB CMake module
  * Adds a FindMenuCache CMake module
  * Adds a FindFm CMake module
  * move project name to top
  * Fixes the name of package config file name in superbuild mode

0.2.0 / 2016-10-30
==================

  * Release 0.2.0: Update changelog
  * Bump minor to 2 (#7)
  * Added Qt5Core to README.md (#6)
  * Adds an LXQtConfigVars module
  * Adds superbuild support
  * README.md: Update dependencies

0.1.0 / 2016-10-14
==================

  * Release 0.1.0: Update changelog
  * Adds LXQtCreatePortableHeaders module
  * Authors and license (#2)
  * Add README.md
  * Make lxqt-build-tools be usable

0.0.0 / 2016-09-30
==================

  * Remove cpack (#107)
  * Bump version ot 0.11.0 (#105)
  * Settings: Add "homemade" XDG_CONFIG_DIRS support
  * translations: Change default translations repository
  * Don't lock the screen when it's already locked.
  * LXQtTranslateTs.cmake: Enable PULL_TRANSLATIONS by default
  * LXQtTranslateTs.cmake: Use/update template in remote repo
  * .gitignore: Cleanup
  * build: Use external translations
  * translations: Fix external translations logic
  * translations: Add external translations support
  * Adds CMAKE_INSTALL_PREFIX to the LXQt Theme search paths
  * LXQtCompilerSettings: Add QT_USE_QSTRINGBUILDER to the compile definitions
  * TranslateDesktopTs: Adds UPDATE_OPTIONS options
  * TranslateDesktop: Remove lupdate absolute option
  * TranslateDesktop: Use -a switch in grep commands
  * Use CMAKE_INSTALL_FULL_DATAROOTDIR
  * Add LXQT_GRAPHICS_DIR directory definition
  * Adds FindXdgUserDirs CMake find_package module
  * Release 0.10.0
  * pluginTranslation: removed using QStringLiteral as it was causing SEGFAULT in static finalization time
  * pluginTranslation: add some symbol for referencing (for file to be not stripped out in static linking time)
  * Adds "automatic" plugin translations loader support
  * Rename LxQt to LXQt everywhere
  * Handles CMake policy CMP0063
  * Adds an lxqt_enable_target_exceptions() CMake function
  * Adds LXQtCompilerSettings CMake module
  * LXQtCreatePkgConfigFile.cmake: Makes install COMPONENT configurable
  * LXQtTranslateTs.cmake Adds install COMPONENT option
  * Adds Runtime install COMPONENT
  * Fixes an SuperBuild/NormalBuild FTBFS
  * CMake: Use CMAKE_AUTOUIC, drop qt5_wrap()
  * CMake: Quote variables
  * CMake: Removes unused variables
  * Removes LIB_INSTALL_DIR CACHE PATH variable
  * Use relative paths in install DESTINATION
  * Marks link libraries as PUBLIC
  * Use target_compile_definitions() instead of add_definitions()
  * Make Release the default build type
  * Simplifies build type matching
  * lxqt-config.cmake.in: Use find_dependency()
  * Updates the build system to the Targets infrastructure
  * Sort .desktop translations. Fix typos in the comments. Remove local dir artifact.
  * pkg-config: Adds Qt5Xdg to the required dependencies
  * Updates lxqt_create_pkgconfig_file()
  * Stops using Qt5Xdg dropped stuff
  * Adds LXQT_DATA_DIR configuration variable

lxqt-panel/panel/translations/lxqt-panel.ts
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1">
<context>
    <name>AddPluginDialog</name>
    <message>
        <location filename="../config/addplugindialog.ui" line="14"/>
        <source>Add Plugins</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/addplugindialog.ui" line="22"/>
        <source>Search:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/addplugindialog.ui" line="98"/>
        <source>Add Widget</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/addplugindialog.ui" line="105"/>
        <source>Close</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/addplugindialog.cpp" line="115"/>
        <source>(only one instance can run at a time)</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>ConfigPanelDialog</name>
    <message>
        <location filename="../config/configpaneldialog.cpp" line="31"/>
        <source>Configure Panel</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpaneldialog.cpp" line="38"/>
        <source>Panel</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpaneldialog.cpp" line="42"/>
        <source>Widgets</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>ConfigPanelWidget</name>
    <message>
        <location filename="../config/configpanelwidget.ui" line="20"/>
        <source>Configure panel</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="44"/>
        <source>Size</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="68"/>
        <source>&lt;p&gt;Negative pixel value sets the panel length to that many pixels less than available screen space.&lt;/p&gt;&lt;p/&gt;&lt;p&gt;&lt;i&gt;E.g. &quot;Length&quot; set to -100px, screen size is 1000px, then real panel length will be 900 px.&lt;/i&gt;&lt;/p&gt;</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="81"/>
        <source>Size:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="88"/>
        <source>Length:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="96"/>
        <source>%</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="101"/>
        <source>px</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="109"/>
        <location filename="../config/configpanelwidget.ui" line="153"/>
        <source> px</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="166"/>
        <source>Icon size:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="202"/>
        <source>Alignment &amp;&amp; position</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="250"/>
        <source>A&amp;uto-hide</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="262"/>
        <location filename="../config/configpanelwidget.ui" line="285"/>
        <source>Zero means no animation</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="288"/>
        <location filename="../config/configpanelwidget.ui" line="314"/>
        <source> ms</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="301"/>
        <location filename="../config/configpanelwidget.ui" line="311"/>
        <source>Zero means no delay</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="304"/>
        <source>Show with delay:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="327"/>
        <source>Visible thin margin for hidden panel</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="337"/>
        <source>Don&apos;t allow maximized windows go under the panel window</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="340"/>
        <source>Reserve space on display</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="568"/>
        <source>A partial workaround for widget styles that
cannot give a separate theme to the panel.

You might also want to disable:

LXQt Appearance Configuration →
Icons Theme →
Colorize icons based on widget style (palette)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="578"/>
        <source>Override icon &amp;theme</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="593"/>
        <source>Icon theme for panels:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="237"/>
        <source>Position:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="265"/>
        <source>Animation duration:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="208"/>
        <source>Alignment:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="173"/>
        <source>Rows:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="219"/>
        <location filename="../config/configpanelwidget.cpp" line="211"/>
        <source>Left</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="224"/>
        <location filename="../config/configpanelwidget.cpp" line="212"/>
        <location filename="../config/configpanelwidget.cpp" line="218"/>
        <source>Center</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="229"/>
        <location filename="../config/configpanelwidget.cpp" line="213"/>
        <source>Right</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="356"/>
        <source>Custom styling</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="377"/>
        <source>Font color:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="417"/>
        <source>Background color:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="456"/>
        <source>Background opacity:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="488"/>
        <source>&lt;small&gt;Compositing is required for panel transparency.&lt;/small&gt;</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.ui" line="516"/>
        <source>Background image:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.cpp" line="177"/>
        <source>Top of desktop</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.cpp" line="178"/>
        <source>Left of desktop</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.cpp" line="179"/>
        <source>Right of desktop</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.cpp" line="180"/>
        <source>Bottom of desktop</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.cpp" line="189"/>
        <source>Top of desktop %1</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.cpp" line="190"/>
        <source>Left of desktop %1</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.cpp" line="191"/>
        <source>Right of desktop %1</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.cpp" line="192"/>
        <source>Bottom of desktop %1</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.cpp" line="217"/>
        <source>Top</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.cpp" line="219"/>
        <source>Bottom</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.cpp" line="439"/>
        <location filename="../config/configpanelwidget.cpp" line="455"/>
        <source>Pick color</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.cpp" line="473"/>
        <source>Pick image</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpanelwidget.cpp" line="473"/>
        <source>Images (*.png *.gif *.jpg)</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>ConfigPluginsWidget</name>
    <message>
        <location filename="../config/configpluginswidget.ui" line="14"/>
        <source>Configure Plugins</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpluginswidget.ui" line="84"/>
        <source>Note: changes made in this page cannot be reset.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpluginswidget.ui" line="112"/>
        <source>Move up</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpluginswidget.ui" line="115"/>
        <location filename="../config/configpluginswidget.ui" line="129"/>
        <location filename="../config/configpluginswidget.ui" line="150"/>
        <location filename="../config/configpluginswidget.ui" line="164"/>
        <location filename="../config/configpluginswidget.ui" line="185"/>
        <source>...</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpluginswidget.ui" line="126"/>
        <source>Move down</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpluginswidget.ui" line="147"/>
        <source>Add</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpluginswidget.ui" line="161"/>
        <source>Remove</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../config/configpluginswidget.ui" line="182"/>
        <source>Configure</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>LXQtPanel</name>
    <message>
        <location filename="../lxqtpanel.cpp" line="1130"/>
        <location filename="../lxqtpanel.cpp" line="1156"/>
        <source>Panel</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lxqtpanel.cpp" line="1159"/>
        <source>Configure Panel</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lxqtpanel.cpp" line="1164"/>
        <source>Manage Widgets</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lxqtpanel.cpp" line="1170"/>
        <source>Add New Panel</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lxqtpanel.cpp" line="1177"/>
        <source>Remove Panel</source>
        <comment>Menu Item</comment>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lxqtpanel.cpp" line="1182"/>
        <source>Lock This Panel</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lxqtpanel.cpp" line="1338"/>
        <source>Remove Panel</source>
        <comment>Dialog Title</comment>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lxqtpanel.cpp" line="1339"/>
        <source>Removing a panel can not be undone.
Do you want to remove this panel?</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>Plugin</name>
    <message>
        <location filename="../plugin.cpp" line="436"/>
        <source>Configure &quot;%1&quot;</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../plugin.cpp" line="441"/>
        <source>Move &quot;%1&quot;</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../plugin.cpp" line="449"/>
        <source>Remove &quot;%1&quot;</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>main</name>
    <message>
        <location filename="../lxqtpanelapplication.cpp" line="95"/>
        <source>Use alternate configuration file.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lxqtpanelapplication.cpp" line="96"/>
        <source>Configuration file</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>
